jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -184,7 +186,8 @@ def search(self, error_on_fail=False):
         elif self.catalog_selected == 'Gaia':
             from astroquery.gaia import Gaia, conf
 
-            with conf.set_temp("ROW_LIMIT", self.max_gaia_sources):
+            with conf.set_temp("ROW_LIMIT", self.max_sources):
+                Gaia.ROW_LIMIT = self.max_sources
```

Comment:
this seems redundant - are both needed?

Reply:
The `with` statement wasn't being respected when setting `max_sources`, tested now with it removed and it looked safe to cut.